### PR TITLE
refactor: profile heatmap tooltip

### PR DESCRIPTION
**Pull request by sshanzel**
In this commit, I have included the customization capability of the tooltip which will also be utilized in the Profile Tooltip.

### CODE REVIEW

**Review by idoshamun — APPROVED** [review]

File: packages/shared/src/components/tooltips/BaseTooltip.tsx [hunk 1]
```diff
@@ -51,14 +58,20 @@ export function BaseTooltip(
       onHide={onHide}
       delay={delay}
       allowHTML
+      animation={!(disableInAnimation && disableOutAnimation)}
```
Comment (rebelchris) [inline packages/shared/src/components/tooltips/BaseTooltip.tsx:61]: It can't ever hit this outAnimation because you return if it exists.

**Review by rebelchris — APPROVED** [review]
Nice work, really like the addition of the container padding and border-radius :)
Just one small note.

File: packages/shared/src/components/tooltips/BaseTooltip.tsx (hunk 1 above)
Comment (sshanzel) [reply packages/shared/src/components/tooltips/BaseTooltip.tsx:61]: Yes, it won't but this is for tippy's internal delay for animation which has to be applied here as well.

**Comment by sshanzel** [comment]
Finished doing sanity checks. WIll be merging now. Thank you both!

File: packages/shared/src/components/tooltips/BaseTooltip.tsx (hunk 1 above)
Comment (sshanzel) [reply packages/shared/src/components/tooltips/BaseTooltip.tsx:61]: That return was to avoid our custom animation.

**Pull request merged**